pkg/scanner/adapter.go

```diff
@@ -35,6 +37,10 @@ func (s *adapter) Scan(req harbor.ScanRequest) (harbor.ScanReport, error) {
 		return harbor.ScanReport{}, fmt.Errorf("preparing layers: %v", err)
 	}
 
+	if len(layers) == 0 {
+		return harbor.ScanReport{}, nil
```

Comment:
The report returned here is parsed by Harbor service so even if there are no vulnerabilities we must return required metadata:

```suggestion
		return harbor.ScanReport{
			GeneratedAt:     time.Now(),
			Scanner:         etc.GetScannerMetadata(),
			Artifact:        req.Artifact,
			Severity:        harbor.SevUnknown,
			Vulnerabilities: []harbor.VulnerabilityItem{},
		}, nil
```